### ISSUE TITLE
Alt text is not descriptive [../pubsub files]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7071]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/pubsub/js/getting-started.mdx:
```
  44  
  45: ![Alt text](/images/create-iot-policy.png)
  46  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/pubsub/js/getting-started.mdx:
```
  45  
  46: ![Alt text](/images/create-iot-policy.png)
  47  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/sdk/pubsub/ios/getting-started.mdx:
```
  70  
  71: ![Alt text](/images/iot_attach_policy.png)
  72  
```

**Comment by vincetran** [comment]
Closing in favor of https://github.com/aws-amplify/docs/issues/7072
